docs/contributing-guide.md

```diff
@@ -0,0 +1,90 @@
+# Contributing guide
```

Comment:
OK
I think github follow simlinks so maybe symlink CONTRIBUTING.md to this file